src/Runner.Worker/ActionRunner.cs

```diff
@@ -135,6 +135,13 @@ public async Task RunAsync()
                 ExecutionContext.SetGitHubContext("event_path", workflowFile);
             }
 
+            // Set GITHUB_ACTION_REPOSITORY if this Action is from a repository
+            if (Action.Reference is Pipelines.RepositoryPathReference repoPathReferenceAction)
```

Comment:
what's the value would be if i use the action from the checkout repo.

ex:
```
- use: ./localaction
  with:
      foo: bar
```

Reply:
It looks like currently they are both set to the empty string. Do you think that's reasonable or should they just not be set at all in that case?

Reply:
maybe just not set.
```
if (Action.Reference is Pipelines.RepositoryPathReference repoPathReferenceAction && 
    !string.Equals(repoPathReferenceAction.RepositoryType, Pipelines.PipelineConstants.SelfAlias, StringComparison.OrdinalIgnoreCase))
{
 // set action_repo/action_ref
}
```